docs/changelog/2013.doc.rst

```diff
@@ -0,0 +1 @@
+Update code style to mention which Python versions code syntax should be used in which files - by :user:`eumiro`.
```

Comment:
```suggestion
Add information why certain files need to stay Python 2 compatible - by :user:`eumiro`.
```

Reply:
I don't think this needs a changelog, considering its not user-facing change 🤔